Tests/Command/GenerateAdminCommandTest.php

```diff
@@ -384,4 +384,108 @@ public function testValidateManagerTypeWithException3()
         $this->setExpectedException('InvalidArgumentException', 'Invalid manager type "baz". Available manager types are "".');
         $this->command->validateManagerType('baz');
     }
+
+    /**
+     * @group admin-gen
+     */
+    public function testAnswerUpdateServicesWithNo()
+    {
+        $this->container->set('sonata.admin.manager.foo', $this->getMock('Sonata\AdminBundle\Model\ModelManagerInterface'));
+
+        $modelEntity = 'Sonata\AdminBundle\Tests\Fixtures\Bundle\Entity\Foo';
+
+        $command = $this->application->find('sonata:admin:generate');
+
+        if (class_exists('Sensio\Bundle\GeneratorBundle\Command\Helper\DialogHelper')) {
```

Comment:
Please add a `NEXT_MAJOR` explaining when we can get rid of the condition. See CONTRIBUTING.md